scanapp/views.py

```diff
@@ -152,6 +154,19 @@ def post(self, request, *args, **kwargs):
             scan_start_time = timezone.now()
             git_url_parser = GitURL(url)
 
+            allowed_exts = ('zip', 'tar', 'tar.gz', 'rar', 'tgz', 'tar.Z', 'tar.bz2',
+                            'tbz2', 'tar.lzma', 'tlz', 'gz')
+            url_parse = urlparse(url)
+
+            is_zip_url = False
+
+            try:
+                for i in allowed_exts:
```

Comment:
We may have some code that identify whether or not files are archives or not. I will ask @pombredanne